Rakefile

```diff
@@ -7,13 +7,43 @@ require 'appraisal'
 Rake::TestTask.new(:test) do |t|
   t.libs << %w(test lib)
   t.test_files = FileList['test/**/*_test.rb'].reject do |path|
-    path.include?('rails') || path.include?('benchmark')
+    path.include?('contrib') ||
+      path.include?('benchmark') ||
+      path.include?('redis') ||
+      path.include?('monkey_test.rb') ||
+      path.include?('autopatch_test.rb')
   end
 end
 
 Rake::TestTask.new(:rails) do |t|
   t.libs << %w(test lib)
-  t.test_files = FileList['test/contrib/rails/**/*_test.rb']
+  t.test_files = FileList['test/contrib/rails/**/*_test.rb'].reject do |path|
+    path.include?('redis')
+  end
+end
+
+Rake::TestTask.new(:railsredis) do |t|
+  t.libs << %w(test lib)
+  t.test_files = FileList['test/contrib/rails/**/*redis*_test.rb']
```

Comment:
one day we should refactor our ``Rakefile`` so that it's easier to have a global ``rake rails`` instead of various declinations